services/web/server/src/simcore_service_webserver/projects/projects_service.py

```diff
@@ -971,13 +975,22 @@ async def update_project_node_state(
         permission="write",  # NOTE: MD: before only read was sufficient, double check this
     )
 
+    # TODO: delete this once workbench is removed from the projects table
```

Comment:
Please create an issue and link it here with a NOTE.
We don't want TODOs, unless you are working on it right away?